### ISSUE TITLE
CREATE_FAILED: HttpHandleRequestLambdaFunction

### ISSUE THREAD

**Issue by ersanir** [localstack/localstack-demo#69]
Hi,

I am seeing below error when I try to run the `make deploy` command.

Error:
CREATE_FAILED: HttpHandleRequestLambdaFunction (AWS::Lambda::Function)
Waiter FunctionActiveV2 failed: Waiter encountered a terminal failure state: For expression "Configuration.State" we matched expected path: "Failed"

Can someone please help me resolve this ?

**Comment by ersanir** [comment]
<img width="1241" alt="image" src="https://github.com/localstack/localstack-demo/assets/17056055/0e7d4b40-0d39-4560-ba28-314914b8f697">

**Comment by Jspascal** [comment]
Facing the same issue, did you find a solution?

**Comment by ersanir** [comment]
Yes! I had to start up local stack using the docker-compose.yml file. The docker-compose file for starting SNS service looks like this 
`version: "3.8"

services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    ports:
      - "4566:4566"            # LocalStack Gateway
    environment:
      - DEFAULT_REGION=us-east-1
      - SERVICES=sns
      - DEBUG=1
      - DOCKER_HOST=unix:///var/run/docker.sock
    volumes:
      - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
`
Then I started up localstack using the command `docker-compose up` on my mac.

If you are trying to start up this demo app, you need to add whatever services you plan on using to the 'SERIVCES' parameter on the docker-compose.yml and then start up localstack using the 'docker-compose' file. Then try deploying this app
